app/decorators/event_decorator.rb

```diff
@@ -8,6 +8,10 @@ def event_type
   end
 
   def happened_at
-    EasternTimePresenter.new(event.created_at).to_s
+    event.created_at
+  end
+
+  def happened_at_in_words
```

Comment:
Not sure if there is a better name besides `happened_at_in_words`. In naming this, I tried to draw upon Rails conventions: `distance_of_time_in_words` and `time_ago_in_words`, but these methods are not completely analogous. Open to alternative naming suggestions.